src/components/Checkbox/Checkbox.scss

```diff
@@ -5,199 +5,199 @@
 //
 // Office UI Fabric
 // --------------------------------------------------
-// Choice fields (radio buttons and checkboxes) styles
-
-// Unselected, radio button (default)
-.ms-ChoiceField {
-  @include ms-font-m;
-  @include ms-u-normalize;
+// Checkbox styles
+
+$ms-checkbox-field-size: 20px;
+$ms-checkbox-transition-duration: 200ms;
+$ms-checkbox-transition-timing: cubic-bezier(.4, 0, .23, 1);
+
+.ms-Checkbox {
+  box-sizing: border-box;
+  color: $ms-color-neutralPrimary;
+  font-family: $ms-font-family-base;
+  font-size: $ms-font-size-m;
+  font-weight: $ms-font-weight-regular;
   min-height: 36px;
+  border: 1px solid transparent;
   position: relative;
+  @include padding-left(8px);
 
   .ms-Label {
     font-size: $ms-font-size-m;
-    padding: 0 0 0 26px;
+    @include padding(0, 0, 0, 26px);
+    display: inline-block;
   }
 }
 
-//== State: Disabled choicefield
-.ms-ChoiceField-input:disabled {
-  + .ms-ChoiceField-field {
-    pointer-events: none;
-    cursor: default;
-
-    &:before {
-      background-color: $ms-color-neutralTertiaryAlt;
-      color: $ms-color-neutralTertiaryAlt;
-    }
-
-    &:after {
-      border-color: $ms-color-neutralLight;
-    }
-
-    .ms-Label {
-      color: $ms-color-neutralTertiary;
-    }
-
-    @media screen and (-ms-high-contrast: active) {
-      &:before {
-        background-color: $ms-color-contrastBlackDisabled;
-        color: $ms-color-contrastBlackDisabled;
-      }
-
-      &:after {
-        border-color: $ms-color-contrastBlackDisabled;
-      }
-
-      .ms-Label {
-        color: $ms-color-contrastBlackDisabled;
-      }
-    }
-
-    @media screen and (-ms-high-contrast: black-on-white) {
-      &:before {
-        background-color: $ms-color-contrastWhiteDisabled;
-        color: $ms-color-contrastWhiteDisabled;
-      }
-
-      &:after {
-        border-color: $ms-color-contrastWhiteDisabled;
-      }
+// The hidden input
```

Comment:
Yes, we do focus on it. When it's in focus a class 'is-inFocus' is added to outer div. I'm afraid by removing it completely I might break it in any of those humongous number of browsers available, as they are all trained to focus on Input by default. I might have to add something like tabIndex="0" to get focus in that case.